Subject: [PATCH] qca-hostap: increase eapol key timeout

Workaround in hostap increasing the EAPOL key timeout to
accommodate the late reception of EAPO key packet from hardware.

--- a/src/ap/wpa_auth.c
+++ b/src/ap/wpa_auth.c
@@ -68,7 +68,7 @@ static void wpa_group_put(struct wpa_authenticator *wpa_auth,
 static int ieee80211w_kde_len(struct wpa_state_machine *sm);
 static u8 * ieee80211w_kde_add(struct wpa_state_machine *sm, u8 *pos);

-static const u32 eapol_key_timeout_first = 100; /* ms */
+static const u32 eapol_key_timeout_first = 200; /* ms */
 static const u32 eapol_key_timeout_subseq = 1000; /* ms */
 static const u32 eapol_key_timeout_first_group = 500; /* ms */
 static const u32 eapol_key_timeout_no_retrans = 4000; /* ms */
